src/timequeue.c

```diff
@@ -30,43 +30,71 @@
 #include "timequeue.h"
 #include "tune.h"
 
-#define TQ_MUF_TYP 0
-#define TQ_MPI_TYP 1
+/*
+ * TODO: Figure out what "TQ_MUF_TREAD" is.  As far as I can tell,
+ *       we never set it.  If it is unused, we should remove the reference
+ *       to it -- it'll simplify a lot of if statements.
+ */
```

Comment:
Guessing here, but “timed read”? TREAD is an in-server define,  it it may have been different at one time. Agreed we should clean it up if unused.

Reply:
It *is* used I think ... maybe ... It's a very twisty code path.  :)